packages/compass/src/app/utils/telemetry.ts

```diff
@@ -81,20 +81,41 @@ async function getHostInformation(
   };
 }
 
+type ExtraConnectionData = {
+  auth_type: string;
+  tunnel: string;
+  is_srv: boolean;
+  is_localhost: boolean;
+  is_atlas_url: boolean;
+  is_do_url: boolean;
+  is_public_cloud?: boolean;
+  public_cloud_name?: string;
+} & CsfleInfo;
+
+type CsfleInfo = {
+  has_kms_aws: boolean;
+  has_kms_gcp: boolean;
+  has_kms_kmip: boolean;
+  has_kms_local: boolean;
+  has_kms_azure: boolean;
+  is_csfle: boolean;
+  has_csfle_schema: boolean;
+};
```

Comment:
In my multiple kms per provider PR, i totally missed the telemetry part. Should we also update this to include the count per provider? probably switching from `has_kms_NAME` to `count_kms_NAME` or just adding another field for count?

Reply:
I'm not sure i feel confident enough to do that here :). How about you and @paula-stacho add that on a different PR, could you create a ticket in 7.3 + 8.0 and make a follow up after this one lands?